Example/Tests/ProfileTests.swift

```diff
@@ -0,0 +1,103 @@
+//
+//  ProfileTests.swift
+//  Blockstack_Example
+//
+//  Created by Shreyas Thiagaraj on 9/17/18.
+//  Copyright © 2018 CocoaPods. All rights reserved.
+//
+
+import Quick
+import Nimble
+import Blockstack
+
+class ProfileSpec: QuickSpec {
+    override func spec() {
+        
+        // For user testing1234.id.blockstack
+        let profileToken = "eyJ0eXAiOiJKV1QiLCJhbGciOiJFUzI1NksifQ.eyJqdGkiOiI2OGExZjZiMy1hNmZlLTQ1ZWEtYTY0OC0xM2JmZDQ1NjMwYjQiLCJpYXQiOiIyMDE4LTEwLTIyVDEyOjM3OjIyLjg5NFoiLCJleHAiOiIyMDE5LTEwLTIyVDEyOjM3OjIyLjg5NFoiLCJzdWJqZWN0Ijp7InB1YmxpY0tleSI6IjAzNjRmYmE0ZDIzZDY0ZTNhMWUxZDJmYjQwZDU4YjMwOWExMTMxYWNiMThiZWNhMjFhNjkzYzliYmQ2YjZjMzE0ZSJ9LCJpc3N1ZXIiOnsicHVibGljS2V5IjoiMDM2NGZiYTRkMjNkNjRlM2ExZTFkMmZiNDBkNThiMzA5YTExMzFhY2IxOGJlY2EyMWE2OTNjOWJiZDZiNmMzMTRlIn0sImNsYWltIjp7IkB0eXBlIjoiUGVyc29uIiwiQGNvbnRleHQiOiJodHRwOi8vc2NoZW1hLm9yZyIsIm5hbWUiOiJUZXN0aW5nMTIzIiwiZGVzY3JpcHRpb24iOiJ0ZXN0ZXIgdGVzdGluZyB0ZXN0cyJ9fQ.BEvMyIU72J8p4xE2Yrcqnq4ZO-DfsPV-jqwCv0-dfYpztr1VtAKrF7MFSS4sMRrSfHdTUEAE1VHe9HLulwBOmA"
+    
+        let profileJSON = """
+              {
+                "token": "eyJ0eXAiOiJKV1QiLCJhbGciOiJFUzI1NksifQ.eyJqdGkiOiI2OGExZjZiMy1hNmZlLTQ1ZWEtYTY0OC0xM2JmZDQ1NjMwYjQiLCJpYXQiOiIyMDE4LTEwLTIyVDEyOjM3OjIyLjg5NFoiLCJleHAiOiIyMDE5LTEwLTIyVDEyOjM3OjIyLjg5NFoiLCJzdWJqZWN0Ijp7InB1YmxpY0tleSI6IjAzNjRmYmE0ZDIzZDY0ZTNhMWUxZDJmYjQwZDU4YjMwOWExMTMxYWNiMThiZWNhMjFhNjkzYzliYmQ2YjZjMzE0ZSJ9LCJpc3N1ZXIiOnsicHVibGljS2V5IjoiMDM2NGZiYTRkMjNkNjRlM2ExZTFkMmZiNDBkNThiMzA5YTExMzFhY2IxOGJlY2EyMWE2OTNjOWJiZDZiNmMzMTRlIn0sImNsYWltIjp7IkB0eXBlIjoiUGVyc29uIiwiQGNvbnRleHQiOiJodHRwOi8vc2NoZW1hLm9yZyIsIm5hbWUiOiJUZXN0aW5nMTIzIiwiZGVzY3JpcHRpb24iOiJ0ZXN0ZXIgdGVzdGluZyB0ZXN0cyJ9fQ.BEvMyIU72J8p4xE2Yrcqnq4ZO-DfsPV-jqwCv0-dfYpztr1VtAKrF7MFSS4sMRrSfHdTUEAE1VHe9HLulwBOmA",
+                "decodedToken": {
+                  "header": {
+                    "typ": "JWT",
+                    "alg": "ES256K"
+                  },
+                  "payload": {
+                    "jti": "68a1f6b3-a6fe-45ea-a648-13bfd45630b4",
+                    "iat": "2018-10-22T12:37:22.894Z",
+                    "exp": "2019-10-22T12:37:22.894Z",
```

Comment:
These tests will start failing in a year - might want to make a note in a comment.